src/video.tsx

```diff
@@ -31,7 +32,8 @@ export interface VideoActions {
   toggleMute: () => void;
 }
 
-export type RenderCallback = (videoElement: ReactNode, state: VideoState, actions: VideoActions, ref: RefObject<SourceElement>) => ReactNode;
+export type RenderCallback = (videoElement: ReactElement<SourceElement>, state: VideoState, actions: VideoActions, ref: RefObject<SourceElement>) => ReactNode;
```

Comment:
Is `ReactElement<SourceElement>` backwards compatible with `ReactNode`?

Reply:
ReactElement is more concreate version of ReactNode. So, yes, I don't think this is a breaking change.